### ISSUE TITLE
Convert SitemapNewsStory to dict

### ISSUE THREAD

**Issue by J535D165** [GateNLP/ultimate-sitemap-parser#18]
`usp.objects.page.SitemapNewsStory` has no attribute `__dict__` because of the usage of slots. Therefore, `vars()` and `__dict__()`  don't work. It would be nice to have an option to parse the attributes to a dict.

```
In [39]: a[-1].__dict__()                                                                                                                                                                                   
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-39-ab06281650cf> in <module>
----> 1 a[-1].__dict__()

AttributeError: 'SitemapPage' object has no attribute '__dict__'

In [40]: vars(a[-1])                                                                                                                                                                                        
---------------------------------------------------------------------------
TypeError                                 Traceback (most recent call last)
<ipython-input-40-ada32e179a17> in <module>
----> 1 vars(a[-1])

TypeError: vars() argument must have __dict__ attribute

``` 

Thanks for this lib. Works well.

**Comment by tanweeralii** [comment]
Helloo @J535D165 , I am a beginner and want to work on this issue. I got what to do but it would be great if you provide me some examples like how to parse. 
Thanks.

**Comment by freddyheppell** [comment]
Fixed in 31dc767, all objects now have a `to_dict()` method
